### DISCUSSION TITLE
How to add DelogoHD in StaxRip?

### DISCUSSION THREAD

**Discussion by kienkzz** [staxrip/staxrip#907] [General]
Hello, I want to add plugin is DelogoHD (GitHub: https://github.com/HomeOfAviSynthPlusEvolution/DelogoHD).
Can I do something to add this? Thanks for reply!

**Comment by jessielw** [comment]
You can generally just add a plug-in to your avisynth dir or staxrip working avisynth dir and use the plugins

**Reply by kienkzz** [reply]
Can you give a clear answer? Each plugin is a different one, not the same in terms of settings (cannot use DelogoHD by other code in Edit)

**Comment by jessielw** [comment]
Download the 64 bit plugin here https://web.archive.org/web/20210919054843if_/https://files.videohelp.com/u/223002/delogo_avsplus.zip place the extracted plugin into your AviSynth plug-ins 64+ auto load directory or the one that is inside stackrip.

Next in the avisynth section in stackrip add the commands

```AddLOGO (clip, string "logofile", string "logoname", int "pos_x", int "pos_y", int "depth", int "yc_y", int "yc_u", int "yc_v", int "start", int "fadein", int "fadeout", int "end", bool "interlaced")
EraseLOGO (clip, string "logofile", string "logoname", int "pos_x", int "pos_y", int "depth", int "yc_y", int "yc_u", int "yc_v", int "start", int "fadein", int "fadeout", int "end", bool "interlaced")```
